distribution/ddf-common/src/main/resources/etc/ws-security/issuer/signature.properties

```diff
@@ -4,4 +4,10 @@ org.apache.ws.security.crypto.merlin.keystore.type=jks
 org.apache.ws.security.crypto.merlin.keystore.password=${javax.net.ssl.keyStorePassword}
 org.apache.ws.security.crypto.merlin.keystore.private.password=${javax.net.ssl.keyStorePassword}
 org.apache.ws.security.crypto.merlin.keystore.alias=${org.codice.ddf.system.hostname}
-org.apache.ws.security.crypto.merlin.keystore.file=${javax.net.ssl.keyStore}
\ No newline at end of file
+org.apache.ws.security.crypto.merlin.keystore.file=${javax.net.ssl.keyStore}
+
+#Truststore properties also defined at http://ws.apache.org/wss4j/config.html
+org.apache.ws.security.crypto.merlin.truststore.type=jks
```

Comment:
❓ What does this do now that the validator isn't using the `signature.properties` file?

Reply:
The validators pass in a system crypto to apache's `SignatureTrustValidator`. If the truststore information is not present, it will use the keystore to validate which we don't want